packages/forms-web-app/src/pages/projects/index/_includes/project-stage.njk

```diff
@@ -41,64 +41,27 @@
 			{% include "./stages/post-decision/description.njk" %}
 		{% endif %}
 
-		<dl class="govuk-summary-list pins-modded-last-item-border-none">
-			<div id="project-stage-{{ projectStages[1] | lower }}" class="govuk-summary-list__row">
-				<dt class="govuk-summary-list__value">
-					{% include "./stages/pre-application/details.njk" %}
-				</dt>
-				<dd class="govuk-summary-list__actions">
-					{{ stageProgressTag( applicationData.status.number, 1) }}
-				</dd>
-			</div>
-			<div id="project-stage-{{ projectStages[2] | lower }}" class="govuk-summary-list__row">
-				<dt class="govuk-summary-list__value">
-					{% include "./stages/acceptance/details.njk" %}
-				</dt>
-				<dd class="govuk-summary-list__actions">
-					{{ stageProgressTag( applicationData.status.number, 2) }}
-				</dd>
-			</div>
-			<div id="project-stage-{{ projectStages[3] | lower }}" class="govuk-summary-list__row">
-				<dt class="govuk-summary-list__value">
-					{% include "./stages/pre-examination/details.njk" %}
-				</dt>
-				<dd class="govuk-summary-list__actions">
-					{{ stageProgressTag( applicationData.status.number, 3) }}
-				</dd>
-			</div>
-			<div id="project-stage-{{ projectStages[4] | lower }}" class="govuk-summary-list__row">
-				<dt class="govuk-summary-list__value">
-					{% include "./stages/examination/details.njk" %}
-				</dt>
-				<dd class="govuk-summary-list__actions">
-					{{ stageProgressTag( applicationData.status.number, 4) }}
-				</dd>
-			</div>
-			<div id="project-stage-{{ projectStages[5] | lower }}" class="govuk-summary-list__row">
-				<dt class="govuk-summary-list__value">
-					{% include "./stages/recommendation/details.njk" %}
-				</dt>
-				<dd class="govuk-summary-list__actions">
-					{{ stageProgressTag( applicationData.status.number, 5) }}
-				</dd>
-			</div>
-			<div id="project-stage-{{ projectStages[6] | lower }}" class="govuk-summary-list__row">
-				<dt class="govuk-summary-list__value">
-					{% include "./stages/decision/details.njk" %}
-				</dt>
-				<dd class="govuk-summary-list__actions">
-					{{ stageProgressTag( applicationData.status.number, 6) }}
-				</dd>
-			</div>
-			<div id="project-stage-{{ projectStages[7] | lower }}" class="govuk-summary-list__row">
-				<dt class="govuk-summary-list__value">
-					{% include "./stages/post-decision/details.njk" %}
-				</dt>
-				<dd class="govuk-summary-list__actions">
-					{{ stageProgressTag( applicationData.status.number, 7) }}
-				</dd>
-			</div>
-		</dl>
+		<div id="project-stage-{{ projectStages[1] | lower }}" class="govuk-!-margin-top-6">
```

Comment:
What's the reason the <dd> tags are removed.  I think this might impact accessibility?
Might be worth checking in WAVE that no errors/warns are introduced.

Reply:
It's been removed as this no longer is a table but a series of details components separated by the <hr>s to give it the same look.
Ran axe dev tools against it with no issues flagged